src/react/databrowser/objects/__tests__/ObjectDelete.test.jsx

```diff
@@ -0,0 +1,67 @@
+import * as s3object from '../../../actions/s3object';
+import { BUCKET_NAME, S3_OBJECT } from '../../../actions/__tests__/utils/testUtil';
+import { List } from 'immutable';
+import ObjectDelete from '../ObjectDelete';
+import React from 'react';
+import { reduxMount } from '../../../utils/test';
+
+describe('ObjectDelete', () => {
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render ObjectDelete component', () => {
+        const { component } = reduxMount(<ObjectDelete bucketName={BUCKET_NAME} toggled={[S3_OBJECT]}
+            prefixWithSlash=''/>, {
+            uiObjects: {
+                showObjectDelete: true,
+            },
+        });
+
+        expect(component.find(ObjectDelete).getDOMNode()).not.toBe(null);
```

Comment:
Out of curiosity, is there a difference between `.getDOMNode()).not.toBe(null);` and `.isEmptyRender()).toBe(true)`?

Reply:
It does the same thing but isEmptyRender is a better way to do it I just forget to update in the ObjectDelete component